packages/framework-integration-tests/integration/providers/aws/utils.ts

```diff
@@ -400,16 +415,12 @@ export async function queryEvents(primaryKey: string, latestFirst = true): Promi
 }
 
 // --- Subscriptions store helpers ---
-export async function subscriptionsTableName(): Promise<string> {
-  const stackName = appStackName()
-
-  return `${stackName}-subscriptions-store`
-}
-
 export async function countSubscriptionsItems(): Promise<number> {
-  const tableName = await subscriptionsTableName()
+  return countTableItems(`${appStackName()}-subscriptions-store`)
+}
```

Comment:
No need to calculate on each call, this can be a const
```suggestion
const countSubscriptionsItems = await countTableItems(`${appStackName()}-subscriptions-store`)
```

Reply:
@NickSeagull mmm I think there is something I'm missing. Each time I call the function `countSubscriptionItems` I need to do the call and get the new count.

I don't understand the suggestion either